docs/users-guide/custom-questions.md

```diff
@@ -0,0 +1,126 @@
+## Creating custom questions with the notebook editor
+
+---
+
+If you have a question that's a bit more involved than a [simple question](04-asking-questions.md), you can create a custom question using the notebook editor. You can get there by clicking the Ask a Question button in the top nav bar and selecting Custom Question. If you're on a question detail page, you can also open up the editor by clicking the icon in the top right of the screen.
```

Comment:
Should add that it's possible to "convert" any existing QB and Simple questions, simply by clicking the Editor-button from the question builder page.